src/picknik_ur_studio_integration_testing/test/CMakeLists.txt

```
@@ -0,0 +1,20 @@
+find_package(rclcpp REQUIRED)
+find_package(ament_cmake_gtest REQUIRED)
+find_package(moveit_studio_agent REQUIRED)
+find_package(moveit_studio_test_utils REQUIRED)
+find_package(ros_testing REQUIRED)
+
+# Integration test for ObjectiveServerNode running with other Agent nodes using real Behaviors
+ament_add_gtest_executable(test_basic_ur5e_objectives test_basic_ur5e_objectives.cpp)
+target_link_libraries(test_basic_ur5e_objectives moveit_studio_agent::do_objective_test_fixture)
+add_ros_test(launch/test_basic_ur5e_objectives.test.py TIMEOUT 120 ARGS "test_binary_dir:=${CMAKE_CURRENT_BINARY_DIR}")
+
+# Integration test for the 'getPlanningGroups' MoveGroup capability.
+ament_add_gtest_executable(test_get_planning_groups  test_get_planning_groups.cpp)
```

ultra nit

```suggestion
ament_add_gtest_executable(test_get_planning_groups test_get_planning_groups.cpp)
```

---

... but also, I'm wondering if we even want to have this integration test here, as it's more "core" to MoveIt Studio. I would say we leave it for now, and I believe the goal for this service is to put it in OSS MoveIt.